src/extensions/lens-renderer-extension.ts

```diff
@@ -44,6 +45,7 @@ export class LensRendererExtension extends LensExtension {
   commands: CommandRegistration[] = [];
   welcomeMenus: WelcomeMenuRegistration[] = [];
   catalogEntityDetailItems: CatalogEntityDetailRegistration[] = [];
+  topBarItems: TopBarRegistration[] = [];
```

Comment:
So you decided not to customize these over the active entity?

Reply:
Active entity can be received inside extension in any place within https://api-docs.k8slens.dev/master/extensions/api/classes/renderer.catalog.catalogentityregistry/#activeentity property.